wgsl/index.bs

```diff
@@ -15289,6 +15289,8 @@ Internet Engineering Steering Group (IESG) for review, approval, and registratio
 :: WebGPU Shading Language (WGSL) is a programming language for GPU
     code to be executed in the context of the WebGPU API. For security
     considerations, see [[!WebGPU]] Section 2.1 Security Considerations.
+    For privacy considerations, see [[WebGPU]] Section 2.2 Privacy
```

Comment:
Security considerations reference as normative, should this also be `
[[!`
?

Reply:
fixed. thanks!